tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_FlushTests.cpp

```diff
@@ -0,0 +1,42 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+
+#if WINOBJC
+#include "CGContextInternal.h"
+DRAW_TEST_F(CGContextFlush, FillFlush, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+    _CGContextPushBeginDraw(context);
+
+    CGContextSetRGBFillColor(context, 1, 0, 0, 1);
+    CGContextFillRect(context, bounds);
+
+    CGContextFlush(context);
+
+    BYTE* dataPtr = static_cast<BYTE*>(CGBitmapContextGetData(context));
+    ASSERT_NE(dataPtr, nullptr);
+
+    EXPECT_EQ(dataPtr[0], 00);
+    EXPECT_EQ(dataPtr[1], 00);
+    EXPECT_EQ(dataPtr[2], 0xff);
+    EXPECT_EQ(dataPtr[3], 0xff);
+
+    _CGContextPopEndDraw(context);
```

Comment:
offline conversation about UT:
this should be a UT, but it can be a drawing test if we look at what happens if we flush+draw+flush+draw.
not 100% okay with testing something out of our control (d2d batching params)